docs/getting-started.adoc

```diff
@@ -54,13 +54,29 @@ $ odo push
 ----
 +
 Your component is now deployed to OpenShift.
-. To access the component, create an OpenShift route and view your
-deployed application using the generated URL:
+. To access the component, create an URL:
 +
 ----
-$ odo url create --open
+$ odo url create --port 8080
 ----
-
++
+. Previous command will make entry in config file, to create URL on cluster:
++
+---
+$ odo push
```

Comment:
What does Push do here? Does it create the URL on the cluster? If so the the numbered step should be either 'create the URL on the cluster' or maybe 'Push the URL to the cluster' since we already created the URL earlier.